PeptideBuilder/PeptideBuilder.py

```diff
@@ -13,18 +13,18 @@
 This file is provided to you under the MIT License.'''
 
 from __future__ import print_function
```

Comment:
Is this line still needed or should it be deleted? I'm not doing a lot of python these days so I'm not sure. If it can be deleted I'd suggest you do it as part of this PR.

Reply:
It's a leftover for Python 2 support and I am happy to remove it in this PR.